packages/react-app-revamp/layouts/LayoutViewContest/Sidebar/index.tsx

```diff
@@ -150,8 +150,8 @@ export const Sidebar = (props: any) => {
                 : "primary"
             }
             disabled={
-              listProposalsIds.length > Object.keys(listProposalsData).length || 
-              currentUserAvailableVotesAmount < amountOfTokensRequiredToSubmitEntry ||
+              listProposalsIds.length > Object.keys(listProposalsData).length ||
```

Comment:
Could we also surface the how many submission tokens a user has here as well as in `JokeDaoV2Dev/packages/react-app-revamp/pages/contest/[chain]/[address]/rules/index.tsx`? More visibility into token (submission vs. voting) information (like having a submission token section and specifying "voting token" in the picture below) would be really helpful with deciphering what is going on.

<img width="539" alt="Screen Shot 2022-08-23 at 3 48 00 PM" src="https://user-images.githubusercontent.com/27874039/186252565-0bc917da-e1bf-4858-997f-215a0f83492c.png">

Reply:
Separated the "Token" section in 2 different sections :heavy_check_mark: 
![Uploading Screenshot 2022-09-05 at 17-36-19 JokeDAO 🃏 An open-source collaborative decision-making platform.png…]()